test_utils.go

```diff
@@ -189,6 +189,138 @@ func (otu *OverflowTestUtils) SendCollectionToTreasury(from string, to string) *
 	return otu
 }
 
+func (otu *OverflowTestUtils) RemoveCollectionFromTreasury(from, treasuryAddr, collectionID string) *OverflowTestUtils {
+
+	//////////////////////////////////////////////
+	// Generate message/signature for signer
+	// msg {hexCollectionID}{blockID}
+	//////////////////////////////////////////////
+
+	src := []byte(collectionID)
```

Comment:
Maybe we should abstract the message/signature generating logic into a function for reuse

Reply:
i'm going to leave this as-is for now. we can revisit after re-auditing all our message formats.